WordPress/Classes/ViewRelated/Notifications/Comment Moderation/CommentModerationOptionsView.swift

```diff
@@ -9,10 +9,10 @@ struct CommentModerationOptionsView: View {
         .spam
     ]
 
-    let onOptionSelected: (Option) -> Void
+    @StateObject private var viewModel: CommentModerationViewModel
```

Comment:
Nit: This view doesn't observe any state, so I believe @StateObject is unnecessary. While keeping it doesn't cause any harm, it suggests that the view is observing some state, which isn't the case.

Reply:
That's true. The change is a remnant from the process of playing with loading state. I'll revert it.

Reply:
Resolved in [e1a9a45](https://github.com/wordpress-mobile/WordPress-iOS/pull/23225/commits/e1a9a453e82f16e7e345b3dcfe9b245a58712e3f)